src/lessopen.rs

```diff
@@ -32,10 +35,18 @@ enum LessOpenKind {
 
 impl LessOpenPreprocessor {
     /// Create a new instance of LessOpenPreprocessor
-    /// Will return Ok if and only if $LESSOPEN is set
+    /// Will return Ok if and only if $LESSOPEN is set and contains exactly one %s
```

Comment:
What do you mean with "exactly one %s"?

Reply:
Let's say you want to use `batpipe` as to preprocess `less` or `bat`'s input.

`batpipe %s` is a valid command for `$LESSOPEN`.

The following are not valid and will result in `less`/`bat` ignoring it and printing a warning:

- `batpipe`
- `batpipe %s %s`
- `batpipe %s %s (...) %s`